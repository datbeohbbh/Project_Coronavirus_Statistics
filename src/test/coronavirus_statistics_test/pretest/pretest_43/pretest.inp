50 100
q 1 44 296
q 1 50 35
q 1 12 486
? 4 3
? 4 2
? 4 2
q 1 46 276
q 1 48 27
q 2 46 36
? 4 4
? 4 4
? 4 2
q 1 33 66
q 1 23 243
q 3 33 3
? 6
q 3 44 30
? 4 4
q 3 48 23
q 1 38 325
? 4 8
q 3 12 361
? 4 7
q 1 22 493
? 6
q 3 48 4
? 7
q 1 42 424
? 4 7
q 1 17 123
? 4 1
? 4 6
? 4 1
? 4 9
? 6
? 7
? 7
? 4 5
q 3 23 170
q 1 28 130
? 4 6
? 6
? 4 7
q 3 17 84
? 6
q 1 50 500
? 4 9
? 7
q 1 8 289
q 3 42 263
q 1 13 347
? 7
q 3 38 26
q 3 23 37
? 4 11
q 2 38 120
? 4 12
q 1 13 209
q 2 28 94
q 1 5 294
q 3 8 141
q 1 24 189
? 4 9
q 1 18 146
? 7
? 4 1
q 1 3 208
? 4 2
q 3 12 98
q 3 46 115
? 4 11
? 6
q 1 10 98
? 4 12
q 1 21 346
q 3 38 105
q 1 6 358
q 1 47 92
? 4 6
? 4 7
? 7
? 7
? 4 5
? 6
? 4 7
? 4 1
? 4 9
? 6
? 4 16
? 7
? 6
? 4 5
? 4 15
? 4 9
? 4 19
? 6
? 4 16
? 6
? 6
? 6