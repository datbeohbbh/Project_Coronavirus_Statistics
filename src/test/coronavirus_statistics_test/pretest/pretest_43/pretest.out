Query ? 4 3 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 486
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 296
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 296
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 296
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 296
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 35
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 36 people

Query ? 4 4 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 240
-> Has recovered : 36
-> Was dead : 0

Query ? 4 8 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 486
-> Has recovered : 0
-> Was dead : 0

Query ? 4 7 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 266
-> Has recovered : 0
-> Was dead : 30

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 36 people

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 361 people

Query ? 4 7 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 325
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 35
-> Has recovered : 0
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 23 
- Information of the city 23 -
-> Number of current patients : 243
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 35
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 42 
- Information of the city 42 -
-> Number of current patients : 424
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 36 people

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 361 people

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 361 people

Query ? 4 5 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 240
-> Has recovered : 36
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 130
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 36 people

Query ? 4 7 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 240
-> Has recovered : 36
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 36 people

Query ? 4 9 :
The answer is the city with the index 42 
- Information of the city 42 -
-> Number of current patients : 424
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 361 people

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 361 people

Query ? 4 11 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 347
-> Has recovered : 0
-> Was dead : 0

Query ? 4 12 :
The answer is the city with the index 22 
- Information of the city 22 -
-> Number of current patients : 493
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 189
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 361 people

Query ? 4 1 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 36
-> Has recovered : 94
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 23 
- Information of the city 23 -
-> Number of current patients : 36
-> Has recovered : 0
-> Was dead : 207

Query ? 4 11 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 189
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 38
Quantity: 120 people

Query ? 4 12 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 189
-> Has recovered : 0
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 74
-> Has recovered : 120
-> Was dead : 131

Query ? 4 7 :
The answer is the city with the index 47 
- Information of the city 47 -
-> Number of current patients : 92
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 459 people

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 459 people

Query ? 4 5 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 63
-> Has recovered : 0
-> Was dead : 3

Query ? 6 :
The city with the highest recovery is city with the index 38
Quantity: 120 people

Query ? 4 7 :
The answer is the city with the index 47 
- Information of the city 47 -
-> Number of current patients : 92
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 27
-> Has recovered : 0
-> Was dead : 459

Query ? 4 9 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 125
-> Has recovered : 36
-> Was dead : 115

Query ? 6 :
The city with the highest recovery is city with the index 38
Quantity: 120 people

Query ? 4 16 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 294
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 459 people

Query ? 6 :
The city with the highest recovery is city with the index 38
Quantity: 120 people

Query ? 4 5 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 63
-> Has recovered : 0
-> Was dead : 3

Query ? 4 15 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 266
-> Has recovered : 0
-> Was dead : 30

Query ? 4 9 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 125
-> Has recovered : 36
-> Was dead : 115

Query ? 4 19 :
The answer is the city with the index 22 
- Information of the city 22 -
-> Number of current patients : 493
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 38
Quantity: 120 people

Query ? 4 16 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 294
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 38
Quantity: 120 people

Query ? 6 :
The city with the highest recovery is city with the index 38
Quantity: 120 people

Query ? 6 :
The city with the highest recovery is city with the index 38
Quantity: 120 people

